www/wp-config.php

```diff
@@ -0,0 +1,83 @@
+<?php
+// ===================================================
+// Load database info and local development parameters
+// ===================================================
+if ( file_exists( dirname( __FILE__ ) . '/local-config.php' ) ) {
+	define( 'WP_LOCAL_DEV', true );
+	include( dirname( __FILE__ ) . '/local-config.php' );
+} else {
+	define( 'WP_LOCAL_DEV', false );
+	define( 'DB_NAME', '%%DB_NAME%%' );
+	define( 'DB_USER', '%%DB_USER%%' );
+	define( 'DB_PASSWORD', '%%DB_PASSWORD%%' );
+	define( 'DB_HOST', '%%DB_HOST%%' ); // Probably 'localhost'
+}
+
+// ========================
+// Custom Content Directory
+// ========================
+define( 'WP_CONTENT_DIR', dirname( __FILE__ ) . '/content' );
+define( 'WP_CONTENT_URL', 'http://' . $_SERVER['HTTP_HOST'] . '/content' );
+
+// ================================================
+// You almost certainly do not want to change these
+// ================================================
+define( 'DB_CHARSET', 'utf8' );
+define( 'DB_COLLATE', '' );
+
+// ==============================================================
+// Salts, for security
+// Grab these from: https://api.wordpress.org/secret-key/1.1/salt
+// ==============================================================
+define( 'AUTH_KEY',         'put your unique phrase here' );
```

Comment:
Done in [`wp-config.php.ctmpl#L43-L50`](https://github.com/misterbisson/triton-wordpress/blob/progress/var/www/html/wp-config.php.ctmpl#L43-L50)